.ci/magician/cmd/generate_downstream.go

```diff
@@ -25,6 +25,7 @@ var gdEnvironmentVariables = [...]string{
 var gdTokenEnvironmentVariables = [...]string{
 	"GITHUB_TOKEN_CLASSIC",
 	"GITHUB_TOKEN_DOWNSTREAMS",
+	"GITHUB_TOKEN",
```

Comment:
Is the logic around this correct? Right now the command will error if _any_ of these are not set, so it won't function as a fallback properly. Should we use the lookup with fallback when looping through this variable instead?

Reply:
the logic is already correct